content/integrations/streaming/bigquery.textile

```diff
@@ -0,0 +1,109 @@
+
+A rule defines what data gets sent, where it goes, and how it's authenticated. For example, you can improve query performance by configuring a rule to stream data from a specific channel and write them into a "partitioned":https://cloud.google.com/bigquery/docs/partitioned-tables table.
+
+h3(#dashboard). Create a rule using the Ably dashboard
+
+The following steps to create a BigQuery rule using the Ably dashboard:
+
+* Log in to the "Ably dashboard":https://ably.com/accounts/any and select the application you want to stream data from.
+* Navigate to the *Integrations* tab.
+* Click *New integration rule*.
+* Select *Firehose*.
+* Choose *BigQuery* from the list of available Firehose integrations.
+* "Configure":#configure the rule settings. Then, click *Create*.
+
+h3(#api-rule). Create a rule using the ABly Control API
+
+The following steps to create a BigQuery rule using the Control API:
+
+* Using the required "rules":/docs/control-api#examples-rules to specify the following parameters:
+** @ruleType@: Set this to "bigquery" to define the rule as a BigQuery integration.
+** destinationTable: Specify the BigQuery table where the data will be stored.
+** @serviceAccountCredentials@: Provide the necessary GCP service account JSON key to authenticate and authorize data insertion.
+** @channelFilter@ (optional): Use a regular expression to apply the rule to specific channels.
+** @format@ (optional): Define the data format based on how you want messages to be structured.
+* Make an HTTP request to the Control API to create the rule.
+
+h2(#configure). Configure BigQuery
+
+Using the Google Cloud "Console":https://cloud.google.com/bigquery/docs/bigquery-web-ui, configure the required BigQuery resources, permissions, and authentication to allow Ably to write data securely to BigQuery.
+
+The following steps configure BigQuery using the Google Cloud Console:
+
+* Create or select a *BigQuery dataset* in the Google Cloud Console.
+* Create a *BigQuery table* in that dataset.
+** Use the "JSON schema":#schema.
+** For large datasets, partition the table by ingestion time, with daily partitioning recommended for optimal performance.
+
+The following steps set up permissions and authentication using the Google Cloud Console:
+
+* Create a Google Cloud Platform (GCP) "service account":https://cloud.google.com/iam/docs/service-accounts-create with the minimal required BigQuery permissions.
+* Grant the service account table-level access control to allow access to the specific table.
+** @bigquery.tables.get@: to read table metadata.
+** @bigquery.tables.updateData@: to insert records.
+* Generate and securely store the *JSON key file* for the service account.
+** Ably requires this key file to authenticate and write data to your table.
+
+h3(#settings). BigQuery configuration options
+
+The following explains the BigQuery configuration options:
+
+|_. Section |_. Purpose |
+| *Source* | Defines the type of event(s) for delivery. |
```

Comment:
I know you've been working on this in a separate PR, but there should be somewhere to link to for the list of these events now.